homework.md

```diff
@@ -56,25 +58,25 @@ When people want to show off code to each other they can put it on Github
 
 What happens when you `fork` a repository?
 
-You delete it
+You have your own version downloaded in github desktop or github
 
 ## Question 10 
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+you take it from github online to github desktop and is available to work on
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you send a file over the internet and is allowed t obe viewed and edited by another person
```

Comment:
You make local code changes and then submit those changes to a remote project maintainer for review before those changes are implemented, or merged. This is called a pull request; you are requesting that someone reviews and approves your changes before they become final.